src/components/core/Heatmap.tsx

```diff
@@ -1,14 +1,20 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
 import * as React from "react";
 import { BaseChart, IChartProps } from "./base/BaseChart";
 import { ChartPropTypes, Requireable } from "../../proptypes/Chart";
+import { getDefaultHeatmapSort } from "../../helpers/sorts";
 
 export { Requireable };
 
 export class Heatmap extends React.PureComponent<IChartProps, null> {
     public static propTypes = ChartPropTypes;
 
     public render() {
-        return <BaseChart type="heatmap" {...this.props} />;
+        const sorts = getDefaultHeatmapSort(this.props.resultSpec);
+        const resultSpecWithSorts = {
+            ...this.props.resultSpec,
+            sorts,
```

Comment:
What if the user specifies their own custom sorts? This would probably override those as well, right?

Reply:
Solved it by handling custom sorts logic in `getDefaultHeatmapSort()`.